lib/meadow/data/works.ex

```diff
@@ -759,4 +780,37 @@ defmodule Meadow.Data.Works do
       {:error, _} -> false
     end
   end
+
+  def iiif_manifest_headers(work_id) do
+    work = Repo.get!(Work, work_id)
+
+    case ExAws.S3.head_object(
+           Config.pyramid_bucket(),
+           iiif_manifest_key(work)
+         )
+         |> ExAws.request() do
+      {:ok, %{headers: headers, status_code: 200}} ->
+        etag =
+          headers
+          |> Enum.into(%{})
+          |> Map.get("ETag")
+          |> Jason.decode!()
```

Comment:
:+1: to using `Jason.decode!` for this. I'd always been stripping the leading and trailing `"` myself, but this is way less prone to error.